misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=hamster-gtk
+Exec=hamster-gtk
+Type=Application
+Terminal=false
+Encoding=UTF-8
+Categories=GTK;GNOME;Utility;
+Version=1.0
+Name[de]=hamster-gtk
```

Comment:
I will remove them for now then. I am not sure about placing them next to the original though. Ive seen both. And in case of several dozen translations having this list before the remaining "metadata" may be inconvenient for manual inspection.